### PR TITLE
Added work on location tool

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Comment by vorburger** [comment]
Cool!! Give me a few days to fix 😆  my dev workstation, so that I can locally test it, and I'll merge this ASAP.

File: blockly/src/App.tsx [hunk 1]
```diff
@@ -40,7 +41,18 @@ const App = () => {
         <>
           <BlocklyWorkspace
             workspace={workspace}
-            onWorkspaceChange={(workspace) => {
+            onWorkspaceChange={(event, workspace) => {
+              if (
+                event.type === Blockly.Events.BLOCK_CHANGE &&
```
Comment (vorburger) [inline blockly/src/App.tsx:46]: Will this also "fire" when ab end-user reconnects and we reload an existing Blocky script/workspace?

**Review by vorburger — COMMENTED** [review]
Qs

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java [hunk 2]
```diff
@@ -73,9 +76,8 @@ public class StaticWebServerVerticle extends AbstractHttpServerVerticle {
             throw new UncheckedIOException(e);
         }
 
-        JWTAuthOptions authConfig = new JWTAuthOptions().addPubSecKey(new PubSecKeyOptions()
-                .setAlgorithm("HS256")
-                .setBuffer("keyboard cat"));
+        JWTAuthOptions authConfig = new JWTAuthOptions().addPubSecKey(
+                new PubSecKeyOptions().setAlgorithm("HS256").setBuffer("keyboard cat"));
```
[the remark this answers is not in the document]
Comment (edewit) [reply web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java:80]: yes, but we don't use it anywhere we only use the public part of the token

File: blockly/src/App.tsx (hunk 1 above)
Comment (edewit) [reply blockly/src/App.tsx:46]: I guess so, but that's functionality we don't have yet

Comment (vorburger) [reply blockly/src/App.tsx:46]: Oh, right; I wasn't sure anymore. I have created #411 re. that.

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (vorburger) [reply web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java:80]: ok

**Review by vorburger — APPROVED** [review]
I now understand that this is just the 1st step to getting the location tool fully working later.

**Pull request merged**